### PR TITLE
Have a more human readable toString method (i.e. for the state graph)

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: src/main/java/de/retest/surili/commons/actions/ChangeValueOfAction.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	public String toString() {
 		return ChangeValueOfAction.class.getSimpleName() //
 				+ "(" //
-				+ "path=" + targetElement.getIdentifyingAttributes().getPath() //
+				+ targetElement.toString() //
```
Comment (beatngu13) [inline src/main/java/de/retest/surili/commons/actions/ChangeValueOfAction.java:29]: `Element#toString()` calls `IdentifyingAttributes#toString()`. If there is no `text`, this only prints the type, which can be quite ambiguous. What I like about the (X)Path approach is that one can easily find a string representation in the browser. What do you think?

Also: You now sometimes use keys / field names, sometimes not. For example:

```
ChangeValueOfAction(input, keysToSend=[foo])
```

To be consistent, I'd rather keep them:


```
ChangeValueOfAction(targetElement=input, keysToSend=[foo])
```

Here and also elsewhere. This also corresponds to what Lombok or most IDEs generate.

File: src/main/java/de/retest/surili/commons/actions/NavigateToAction.java [hunk 2]
```diff
@@ -19,4 +19,9 @@
 		return Optional.empty();
 	}
 
+	@Override
+	public String toString() {
+		return NavigateToAction.class.getSimpleName() + "(" + url + ")";
```
Comment (beatngu13) [inline src/main/java/de/retest/surili/commons/actions/NavigateToAction.java]: To pick up the keys / field names argument from above: Here you override the Lombok `toString()` method just to omit `url=`. Worth it?

Also above, e.g., `NavigateRefreshAction`: Lombok generates `NavigateRefreshAction()`, your override `NavigateRefreshAction`. Worth it?

**Review by beatngu13 — CHANGES_REQUESTED** [review]

File: src/main/java/de/retest/surili/commons/actions/ChangeValueOfAction.java (hunk 1 above)
Comment (roesslerj) [reply src/main/java/de/retest/surili/commons/actions/ChangeValueOfAction.java:29]: Ok

File: src/main/java/de/retest/surili/commons/actions/NavigateToAction.java (hunk 2 above)
Comment (roesslerj) [reply src/main/java/de/retest/surili/commons/actions/NavigateToAction.java]: Thanks, I wasn't aware that `@Value` incorporates `@ToString`.

**Comment by roesslerj** [comment]
Siehe auch https://github.com/retest/recheck/pull/665...

File: src/test/java/de/retest/surili/commons/actions/ChangeValueOfActionTest.java [hunk 3]
```diff
@@ -2,24 +2,30 @@
...
 
 class ChangeValueOfActionTest {
 
 	@Test
 	void toString_should_be_helpful() {
-		final Element targetElement = mock( Element.class, Mockito.RETURNS_DEEP_STUBS );
-		when( targetElement.getIdentifyingAttributes().getPath() ).thenReturn( "/foo[1]/bar[2]" );
+		Collection<Attribute> identifying = IdentifyingAttributes.createList( Path.fromString( "/foo[1]/a[2]" ), "a" );
+		identifying.add( new TextAttribute( "text", "Log In" ) );
+		final Element targetElement = Element.create( "retestId", mock(Element.class), new IdentifyingAttributes( identifying ), new MutableAttributes().immutable() );
```
Comment (beatngu13) [inline src/test/java/de/retest/surili/commons/actions/ChangeValueOfActionTest.java]: Formatter?

```suggestion
		final Element targetElement = Element.create( "retestId", mock( Element.class ), new IdentifyingAttributes( identifying ), new MutableAttributes().immutable() );
```

File: src/test/java/de/retest/surili/commons/actions/ClickOnActionTest.java [hunk 4]
```diff
@@ -2,23 +2,29 @@
...
 
 class ClickOnActionTest {
 
 	@Test
 	void toString_should_be_helpful() {
-		final Element targetElement = mock( Element.class, Mockito.RETURNS_DEEP_STUBS );
-		when( targetElement.getIdentifyingAttributes().getPath() ).thenReturn( "/foo[1]/bar[2]" );
+		Collection<Attribute> identifying = IdentifyingAttributes.createList( Path.fromString( "/foo[1]/a[2]" ), "a" );
+		identifying.add( new TextAttribute( "text", "Log In" ) );
+		final Element targetElement = Element.create( "retestId", mock(Element.class), new IdentifyingAttributes( identifying ), new MutableAttributes().immutable() );
```
Comment (beatngu13) [inline src/test/java/de/retest/surili/commons/actions/ClickOnActionTest.java]: As above:

```suggestion
		final Element targetElement = Element.create( "retestId", mock( Element.class ), new IdentifyingAttributes( identifying ), new MutableAttributes().immutable() );
```

**Review by beatngu13 — CHANGES_REQUESTED** [review]

**Review by beatngu13 — APPROVED** [review]

**Pull request merged**